### PR TITLE
fix: query tab scrollbar padding gap fix

### PR DESCRIPTION
**Pull request by alex-golovanov**
## Description
Remove scrollbar gap in request config.

Fixes [#38135](https://github.com/appsmithorg/appsmith/issues/38135)

## Automation

/ok-to-test tags="@tag.IDE"

### :mag: Cypress test results

> [!TIP]
> 🟢 🟢 🟢 All cypress tests have passed! 🎉 🎉 🎉
> Workflow run: <https://github.com/appsmithorg/appsmith/actions/runs/12491071853>
> Commit: 85399dbbd44aef19ce3da06a8d6f190eed210de5
> <a href="https://internal.appsmith.com/app/cypress-dashboard/rundetails-65890b3c81d7400d08fa9ee5?branch=master&workflowId=12491071853&attempt=1" target="_blank">Cypress dashboard</a>.
> Tags: `@tag.IDE`
> Spec:
> <hr>Wed, 25 Dec 2024 08:47:23 UTC

## Communication
Should the DevRel and Marketing teams inform users about this change?
- [ ] Yes
- [ ] No

## Summary by CodeRabbit

- **New Features**
	- Introduced a tabbed interface in the CommonEditorForm for better content organization.
	- Enhanced usability of the PluginActionForm with scrollable content areas.

- **Bug Fixes**
	- Adjusted tooltip positioning to improve visibility in the EmbeddedDatasourcePathField.

- **Style**
	- Added new styled components for tabs and headers to enhance layout and design.

- **Chores**
	- Deprecated the RequestTabs component in preparation for its removal.

### CODE REVIEW

File: app/client/src/PluginActionEditor/components/PluginActionForm/components/CommonEditorForm/CommonEditorForm.tsx [hunk 1]
```diff
@@ -29,39 +45,89 @@ const CommonEditorForm = (props: Props) => {
...
-        datasourceHeaders={datasourceHeaders}
-        datasourceParams={datasourceParams}
-        formName={props.formName}
-        paginationUiComponent={props.paginationUiComponent}
-        pushFields={props.isChangePermitted}
-        showSettings={false}
-        theme={theme}
-      />
-    </Flex>
+    <Styled.Tabs onValueChange={setCurrentTab} value={currentTab}>
```
Comment (hetunandu) [inline app/client/src/PluginActionEditor/components/PluginActionForm/components/CommonEditorForm/CommonEditorForm.tsx:57]: Screenshot 2024-12-24 at 10 33 32 AM


Seeing this sticky getting cut off when scrolling

**Review by hetunandu — CHANGES_REQUESTED** [review]

File: app/client/src/PluginActionEditor/components/PluginActionForm/components/CommonEditorForm/CommonEditorForm.tsx (hunk 1 above)
Comment (alex-golovanov) [reply app/client/src/PluginActionEditor/components/PluginActionForm/components/CommonEditorForm/CommonEditorForm.tsx:55]: Fixed.

**Review by hetunandu — APPROVED** [review]

**Pull request merged**